#1051 Add setting to display speed in mph

NextSim opened the pull request.

NextSim committed:
#1051 Add setting to display speed in mph

sytelus commented:
This seems like good change. However, can we make it slightly more cleaner and generic? I would suggest we add two settings:

1. speed_unit_factor (defaults to 1)
2. speed_unit_label (defaults to "m/s")

Then folks can change above to km/hr or miles/hr or miles/day whatever else they need for their specific vehicle type.

NextSim commented:
Sounds good

NextSim committed:
#1051 Add generic settings to change speed units and scale factor

Allows the user to specify any units they want and then convert the speed as appropriate.  Defaults to m/s with a scale factor of 1.

NextSim committed:
#1051 Add new settings to documentation

NextSim committed:
Merge branch 'master' into master

NextSim committed:
Merge branch 'master' into master

NextSim commented:
I think this is ready to be merged in.  Is there anything else I need to do?
